addons/website_quote/models/sale_quote.py

```diff
@@ -92,11 +94,15 @@ def _onchange_product_uom(self):
 
     @api.model
     def create(self, values):
+        if values.get('display_type', self.default_get(['display_type'])['display_type']):
+            values.update(product_id=None, price_unit=0, product_uom_qty=0, product_uom_id=None)
```

Comment:
Small note: as above, set False as default value for m2o

Reply:
Changed